controllers/cartController.js

```diff
@@ -2,12 +2,20 @@ const cartService = require("../services/cartService");
 const appError = require('../middlewares/appError');
 
 const getCartLists = async(req, res) => {
-        const userId = req.user.id;
-        if(!userId) throw new appError('KEY_ERROR', 400);
-        const list = await cartService.getCartLists(userId);
-        return res.status(200).json(list);
+    const userId = req.user.id;
+    if(!userId) throw new appError('KEY_ERROR', 400);
+    const list = await cartService.getCartLists(userId);
+    return res.status(200).json(list);
+};
+
+const addProduct = async(req, res) => {
+    const {userId, productId, amount} = req.body;
+    if(!userId || !productId ||!amount) throw new appError('KEY_ERROR', 400);
+    await cartService.addProduct(userId, productId, amount);
+    return res.status(201).json({message : "PRODUCT_ADDED"})
```

Comment:
가독성이 좋지 않습니다! 한 줄 씩 띄워주세요-!
```suggestion
    const {userId, productId, amount} = req.body;
    
    if(!userId || !productId ||!amount) throw new appError('KEY_ERROR', 400);
    
    await cartService.addProduct(userId, productId, amount);
    
    return res.status(201).json({message : "PRODUCT_ADDED"})
```

Reply:
넵!